src/main/java/frc/robot/oi/CommandControllerWrapper.java

```diff
@@ -65,19 +63,19 @@ public double getRightTriggerAxis() {
     }
 
     public double getLeftX() {
-        return xboxController != null ? xboxController.getLeftX() : -ps5Controller.getLeftX();
+        return xboxController != null ? xboxController.getLeftX() : ps5Controller.getLeftX();
```

Comment:
The X/Y report the same pos/neg between the two controller types - I was incorrectly assuming my alliance when driving (rotation was backwards, but correct now)